tempesta_fw/http.c

```diff
@@ -1055,6 +1146,7 @@ tfw_http_resp_process(TfwConnection *conn, struct sk_buff *skb,
 			TFW_INC_STAT_BH(serv.msgs_parserr);
 			return TFW_BLOCK;
 		case TFW_POSTPONE:
+			((TfwHttpResp *)hmresp)->skb_offset = off;
```

Comment:
Looks like ugly workaround. Why not to add `off` and `skb` to the `TfwGState` struct and avoid passing them via the arguments?